xds/internal/xdsclient/transport/transport.go

```diff
@@ -243,13 +253,15 @@ func (t *Transport) SendRequest(url string, resources []string) {
 
 func (t *Transport) newAggregatedDiscoveryServiceStream(ctx context.Context, cc *grpc.ClientConn) (adsStream, error) {
 	// The transport retries the stream with an exponential backoff whenever the
-	// stream breaks. But if the channel is broken, we don't want the backoff
-	// logic to continuously retry the stream. Setting WaitForReady() blocks the
-	// stream creation until the channel is READY.
-	//
-	// TODO(easwars): Make changes required to comply with A57:
-	// https://github.com/grpc/proposal/blob/master/A57-xds-client-failure-mode-behavior.md
-	return v3adsgrpc.NewAggregatedDiscoveryServiceClient(cc).StreamAggregatedResources(ctx, grpc.WaitForReady(true))
+	// stream breaks or if the channel is broken.
+	return v3adsgrpc.NewAggregatedDiscoveryServiceClient(cc).StreamAggregatedResources(ctx)
+}
+
+// UpdateChannelInfo is the type for resource request updates pushed to the top-level
```

Comment:
How about renaming this as `SendInfo` or `ResourceSendInfo` and changing the comments as follows:

```golang
// ResourceSendInfo wraps the names and url of resources successfully
// sent to the management server. This is used by the `authority` type
// to start/stop the watch timer associated with every resource.
```

Also, there doesn't seem to be any `UpdateCh` anymore. So, we should remove any mentions of that from the comments.